content/docs/tutorials/introduction.md

```diff
@@ -154,13 +153,17 @@ Contributions are welcome! Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md)
 * Modsecurity team for creating ModSecurity
 * OWASP Coreruleset team for the CRS and their help
 
-### Companies using Coraza
+### Companies/Products using Coraza
```

Comment:
Maybe we want to add coraza cc @M4tteoP as per https://docs.tetrate.io/envoy-gateway/howto/coraza